.github/workflows/e2e.yml

```diff
@@ -45,45 +45,48 @@ jobs:
       SYSTEM_TEST_JS_STELLAR_SDK_GH_REF: 
 
       # the version of rs-stellar-xdr to use for quickstart
-      SYSTEM_TEST_RS_XDR_GIT_REF: v20.0.2
+      SYSTEM_TEST_RS_XDR_GIT_REF: v20.1.0
 
       # system test will build quickstart image internally to use for running the service stack
       # configured in standalone network mode(core, rpc)
-      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#412bb828ddb4a93745227ab5ad97c623d43f3a5f
+      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#eeb44ee99c564275654800edea2e426d73e966b8
 
       # triggers system test to log out details from quickstart's logs and test steps
       SYSTEM_TEST_VERBOSE_OUTPUT: "true"
 
       # the soroban test cases will compile various contracts from the examples repo
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "v20.0.0"
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "v20.2.0"
       SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_REPO: "https://github.com/stellar/soroban-examples.git"
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         name: checkout system-test
         with:
           repository: stellar/system-test
           ref: ${{ env.SYSTEM_TEST_GIT_REF }}
           path: system-test
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
I don't think need to checkout local copy of soroban-tools, cli refs to gh workspace directory, and rpc refs soroban-rpc repo now

Reply:
I think it's needed for pathing? unsure though

Reply:
system-test make/build will not be referring to the soroban-tools path, it resolves rpc and cli source paths from the urls or local paths set on  `SYSTEM_TEST_SOROBAN_CLI_REF` and `SYSTEM_TEST_SOROBAN_RPC_REF`

Reply:
does soroban-cli need to use a pinned ref to soroban-rpc for testing purposes? [current config on SYSTEM_TEST_SOROBAN_RPC_REF](https://github.com/stellar/soroban-cli/pull/1208/files#diff-3e103440521ada06efd263ae09b259e5507e4b8f7408308dc227621ad9efa31eR22) is not spec'ing a git ref on the repo url, using mainline/master ref then.